Homework_8/Task_1.py

```diff
@@ -0,0 +1,16 @@
+
+roles = {'admin': 'Vitalii', 'maintainer': 'Josh', 'manager': 'Will', 'developer': 'Steve'}
```

Comment:
values should be lists

Reply:
is it should be couple names for one key?

Reply:
Fixed it and added